xml/System.Formats.Tar/TarWriter.xml

```diff
@@ -74,9 +117,9 @@
 
 ## Remarks
 
-If the selected `archiveFormat` is <xref:System.Formats.Tar.TarFormat.Pax>, no Global Extended Attributes entry is written. To write a PAX archive with a Global Extended Attributes entry inserted at the beginning of the archive, use the <xref:System.Formats.Tar.TarWriter.%23ctor(System.IO.Stream,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.String}},System.Boolean)> constructor instead.
+If the selected `archiveFormat` is <xref:System.Formats.Tar.TarEntryFormat.Pax>, no Global Extended Attributes entry is written. To write a PAX archive with a Global Extended Attributes entry inserted at the beginning of the archive, use the <xref:System.Formats.Tar.TarWriter.%23ctor(System.IO.Stream,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.String}},System.Boolean)> constructor instead.
```

Comment:
This comment is now obsolete. We don't have a constructor that takes a Global Extended Attributes dictionary. Now the Global Extended Attributes are described by their own class.

```suggestion
```